src/anders_mathiesen_ex/ex01/tidy_code.py

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
+
+from random import randint
+
+__author__ = 'Anders Mathiesen'
+__email__ = 'andermat@nmbu.no'
+
+
+def get_guess_bigger_then_or_lik_1():
+    guess = 0
+    while guess < 1:
+        guess = input('Your guess bigger then or lik 1: ')
+        try:
+            guess = int(guess)
+        except:
+            print('Enter a number')
+            guess = 0
+    return guess
+
+
+def get_random_number():
+    return randint(1, 6) + randint(1, 6)
+
+
+def check_if_guess_is_correct(random_number, guess):
```

Comment:
conventions is to write func names that return booleans as a "questions". i.e. just is_correct. In this case though, `is_equal()` would be much better, as that is what the function does irrespective of the rest of the script.